### DISCUSSION TITLE
Cannot Pull Specific Docker Image Tag

### DISCUSSION THREAD

**Discussion by barrelmaker97** [mbround18/valheim-docker#708] [Help!] [answered]
Hi, I like to use specific Docker image tags to make sure that my deployments are exactly as expected. I saw that a new version was release, version 2.1.2, but cannot seem to pull the Docker image, and looking at Docker Hub it does not appear to exist:
```
barrelmaker@zeus:~$ docker pull docker.io/mbround18/valheim:2.1.2
Error response from daemon: manifest for mbround18/valheim:2.1.2 not found: manifest unknown: manifest unknown
```
Is this intentional? I would like to use specific semver tags if possible. I noticed that the 2.1 tag still works.

**Comment by mbround18** [comment]
@barrelmaker97 

![image](https://github.com/mbround18/valheim-docker/assets/12646562/89c5537d-15f9-41fb-a159-ca9d52fbce16) i think the version failed to publish
Oop i do see failures

![image](https://github.com/mbround18/valheim-docker/assets/12646562/204dd6c7-2d04-4ba3-b6e5-79162f2dcca4)

ill dive into it shortly

**Reply by mbround18** [reply, marked as answer]
@barrelmaker97 fixed https://github.com/mbround18/valheim-docker/actions/runs/5579631285/jobs/10195443878

https://hub.docker.com/layers/mbround18/valheim/2.1.2/images/sha256-28a0d207da714dbb5198ef930010f29e7a7f88ed87c50e5bd2ca9d4767c5ba99?context=explore

**Reply by barrelmaker97** [reply]
Thanks! 😁
